bin/run_web.sh

```diff
@@ -10,15 +10,13 @@ set -euxo pipefail
 
 PORT=${PORT:-"8000"}
 GUNICORN_WORKERS=${GUNICORN_WORKERS:-"1"}
-GUNICORN_WORKER_CONNECTIONS=${GUNICORN_WORKER_CONNECTIONS:-"4"}
```

Comment:
`GUNICORN_WORKER_CONNECTIONS` sets the number of coroutines for the gevent worker. We switched to the sync worker when we redid the crashmover and dropped gevent.

Reply:
Did a global search, and I also see it mentioned (in the stdout example) in `dev.rst` as well if you want to remove all references.